geps/gep-713/index.md

```diff
@@ -1,1157 +1,747 @@
+The hierarchical relationships of the object that are targeted by meta resources – whether corresponding to their parent contexts or in relation to their inner sections – may yield indirect conflicts of specs (conflicting intents). Meta resource kinds that allow for their instances to target at multiple levels of a hierarchy of resource kinds (e.g. Gateway API `Gateway` and `HTTPRoute` kinds), entire resources as well as sections of a resource, or resources and filtered contexts of these resource kind (e.g. with `gatewayClassName`) will often generate cases where the behavior specified by the meta resource either is fully enforced or partially enforced, either honored or overridden by another.
 
-**Ana**: _Dude. I don’t even know how to._ 😂
+Meta resource CRDs MUST clearly define the hierarchy of target resources they have effects upon, as well as the semantics of targeting each kind of resource in the hierarchy. Moreover, lower levels in the hierarchy *inherit* the definitions applied at the higher levels, in such a way that higher level rules may be understood as having an “umbrella effect” over everything under that level.
 
-**Chihiro**: _You just attach a RetryPolicy to your HTTPRoute._
+E.g., in Gateway API’s hierarchy of network resources for the ingress use case `GatewayClass` \> `Gateway` \> `HTTPRoute` \> `Backend`, a meta resource that attaches to a `GatewayClass` object, if defined as a meta resource kind ultimately to augment the behavior of `HTTPRoute` objects, affects all `Gateways` under the `GatewayClass`, as well as all `HTTPRoutes` under those `Gateways`. Any other instance of this meta resource kind targeting a lower level than the `GatewayClass` (e.g. `Gateway` or `HTTPRoute`) should be be treated as a conflict against the higher level meta resource spec, for the specific scope (“context”) of the subset of the hierarchy it attaches to. This conflict MUST be resolved according to some defined *merge strategy*.
 
-**Ana**: _Nope. Definitely didn’t do that._
+### Declared targets versus Effective targets
 
-She types `kubectl get retrypolicy -n baker` and gets a permission error.
+A kind specified in the target reference of a meta resource can be the actual kind of object whose behavior the meta resource intends to augment or an indirection to targeting other kinds the object is hierarchically related to.
 
-**Ana**: _Huh, I actually don’t have permissions for RetryPolicy._ 🤔
+E.g. targeting a Gateway API `Gateway` object with a meta resource can be a way to augment the behavior of the `Gateway` object itself (e.g. reconcile cloud infrastructure provider settings from the spec declared by the `Gateway` according the rules specified by the meta resource attached to the `Gateway`) or a means to augment the behavior of all `HTTPRoute` objects attached to the `Gateway` (in a way that every new `HTTPRoute` that gets created or modified so it enters the context of the `Gateway` is automatically put in the scope of the meta resource.)
```

Comment:
I think this needs some breaking up, as it's a bit difficult to parse as it stands.
```suggestion
E.g. targeting a Gateway API `Gateway` object with a meta resource can be:
* a way to augment the behavior of the `Gateway` object itself (e.g. reconcile cloud infrastructure provider settings from the spec declared by the `Gateway` according the rules specified by the meta resource attached to the `Gateway`) or
* a means to augment the behavior of all `HTTPRoute` objects attached to the `Gateway` (in a way that every new `HTTPRoute` that gets created or modified so it enters the context of the `Gateway` is automatically put in the scope of the meta resource.)
```